### DISCUSSION TITLE
How to include a nested relation?

### DISCUSSION THREAD

**Discussion by ilijavuk** [ppetzold/nestjs-paginate#808] [Q&A] [answered]
Hello, I've got the following case:

User entity has a ManyToOne relationship with a City
A City has a ManyToOne relationship with a Country

How can I include both the City and it's Country in the User pagination config?
This doesn't seem to work:
`
  relations: [
    'city',
    'city.country'
  ]`

**Comment by ppetzold** [comment, marked as answer]
try `{ city: { country: true }}` for nested relations

**Reply by ilijavuk** [reply]
Thank you, this works!
